dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/srt/ISRTServletRequest.java

```diff
@@ -23,4 +25,13 @@ public interface ISRTServletRequest {
     void setSSLAttributesInRequest(HttpServletRequest httpServletReq, String cs);
     
     String getCipherSuite();
+
+    String getHeader(HttpHeaderKeys headerKey);
```

Comment:
is the intention of these new methods to make it clear that we are using our original request object to look for our predefined headerKey ?

Reply:
The point is not to have to look up the HttpHeaderKeys object from a String, i.e. findKey does not need to be called.